### ISSUE TITLE
Objects and Object Constructors: <Change link to 'this' question in Knowledge check to reflect Assignment article>

### ISSUE THREAD

**Issue by Meltasy** [TheOdinProject/curriculum#28369]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

In the Assignment section, #2, there is a link to the following article about using 'this': https://www.javascripttutorial.net/javascript-this/. Then in the Knowledge check, the link in the question is for a different article: https://dmitripavlutin.com/gentle-explanation-of-this-in-javascript/. I'd suggest changing the link in the Knowledge check so that it directs you to the same link as used in the Assignment section. 

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-objects-and-object-constructors

### (Optional) Discord Name

Meltasy

**Comment by MaoShizhong** [comment]
Thanks @Meltasy. Looks like it was just missed when the assignment was updated.

## Acceptance criteria

Please comment below if you would like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer. 

- [ ] Change knowledge check href for "How does this behave in different situations?” to `https://www.javascripttutorial.net/javascript-this/`

**Comment by mathdebate09** [comment]
Hey Mao, I can work this fix up :)

**Comment by MaoShizhong** [comment]
Go for it
